lib/content/utilities.ex

```diff
@@ -69,7 +69,7 @@ defmodule Content.Utilities do
   def destination_for_prediction("Green-E", 0, _), do: {:ok, :heath_street}
   def destination_for_prediction("Green-B", 1, _), do: {:ok, :government_center}
   def destination_for_prediction("Green-C", 1, _), do: {:ok, :government_center}
-  def destination_for_prediction("Green-D", 1, _), do: {:ok, :north_station}
+  def destination_for_prediction("Green-D", 1, _), do: {:ok, :union_square}
```

Comment:
Snuck in this small change here. This clause is mainly a catch-all for D Line predictions and needed to be updated.